src/pages/content/principles.mdx

```diff
@@ -0,0 +1,48 @@
+---
+title: Principles
+description: >
+  The domain of UX content strategy is all the language in a user interface (UI). UX content
+  strategists design with words to create experiences that are clear, consistent, and inclusive.
+---
+
+| Be clear                                                                        | Be consistent                                                            | Be inclusive                                                                      |
+| ------------------------------------------------------------------------------- | ------------------------------------------------------------------------ | --------------------------------------------------------------------------------- |
+| Clear language helps people feel safe about their actions in a user experience. | Consistent language lowers the friction of learning and navigating a UI. | Inclusive language makes an experience usable for the broadest possible audience. |
```

Comment:
Using minimal MD would make this more readable in context. See https://www.markdownguide.org/extended-syntax/#tables for details. Ex:

```md
| Be clear | Be consistent | Be inclusive |
| --- | --- | --- |
| Clear language ... | Consistent language ... | Inclusive language ... |
```

Reply:
@allisonacs your baseline designs didn't express spacing for tables. Is an update needed?

Reply:
I think my `yarn format` [command might be forcing that formatting](https://share.getcloudapp.com/2Nu5PBOn). Should I try something else?

Reply:
Nah, forget about it. The binary nature of `prettier` is making this impossible to control 🙁